app/mailers/csv_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CsvMailer < ApplicationMailer
+  def created_email(csv_export)
```

Comment:
all methods are emails -> no need for the _email suffix (like naming controller methods `def show_action`)